modin/pandas/dataframe.py

```diff
@@ -795,25 +795,40 @@ def equals(self, other):  # noqa: PR01, RT01, D200
             and self.eq(other).all().all()
         )
 
-    def _update_var_dicts_in_kwargs(self, expr, kwargs):
+    def _update_var_dicts_in_kwargs(self, expr, level: int, kwargs):
         """
         Copy variables with "@" prefix in `local_dict` and `global_dict` keys of kwargs.
 
+        This function exists because we neeed to infer local and variables
```

Comment:
should this say `local and *global* variables` here? it feels like it's missing a word.